### ISSUE TITLE
'High Contrast Black' theme Select states WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1015]
**Describe the bug**
The Select components active and hover state colors contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/select.html
2. Check the Select button and list's active and hover states' color contrast 

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81756977-27045780-9472-11ea-8e84-b688d84082aa.png)
![image](https://user-images.githubusercontent.com/43764832/81756110-a93f4c80-946f-11ea-96b9-778183110978.png)
![image](https://user-images.githubusercontent.com/43764832/81753968-da1c8300-9469-11ea-8acc-3bd4f2de9a25.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)
